### PR TITLE
9733 Added configurable HTTP timeout

### PR DESCRIPTION
**Pull request by Sergio-Mira**
https://swrvedev.jira.com/browse/SWRVE-9733

### CODE REVIEW

**Comment by Sergio-Mira** [comment]
@dominicmarmionswrve @shanetmoore Could you have a look?

File: SwrveCommonSDK/src/main/java/com/swrve/sdk/config/SwrveConfigBase.java [hunk 1]
```diff
@@ -121,6 +121,11 @@
     private int defaultBackgroundColor = Color.TRANSPARENT;
 
     /**
+     * HTTP timeout used when contacting the Swrve APIs.
+     */
+    private int httpTimeout = 60000;
+
```
Comment (dominicmarmionswrve) [inline SwrveCommonSDK/src/main/java/com/swrve/sdk/config/SwrveConfigBase.java]: Might be better to use constants in the future (and longs), especially for milliseconds. Can be error prone. eg:
private static final long SECOND = 1000;
private static final long httpTimeout = SECOND*60;

**Comment by dominicmarmionswrve** [comment]
:rocket:

File: SwrveCommonSDK/src/main/java/com/swrve/sdk/config/SwrveConfigBase.java (hunk 1 above)
Comment (dominicmarmionswrve) [inline SwrveCommonSDK/src/main/java/com/swrve/sdk/config/SwrveConfigBase.java]: Actually, better the constant is SECOND_MS to indicate its milliseconds.

**Pull request merged**